src/literature/components/ReferenceItem.jsx

```diff
@@ -1,51 +1,62 @@
 import React, { Component, Fragment } from 'react';
 import PropTypes from 'prop-types';
 import { Map } from 'immutable';
-import { List } from 'antd';
+import { List, Row, Col } from 'antd';
 import { Link } from 'react-router-dom';
 
 import AuthorList from '../../common/components/AuthorList';
 import Latex from '../../common/components/Latex';
 import PublicationInfoList from '../../common/components/PublicationInfoList';
 
 class ReferenceItem extends Component {
+  static renderLabel(reference) {
+    const label = reference.get('label');
+    const labelDisplay = label ? `[${label}] ` : null;
+    return labelDisplay;
+  }
+
   static renderTitle(reference) {
     const recordId = reference.get('control_number');
     const title = reference.getIn(['titles', 0, 'title'], '');
-    const label = reference.get('label');
-
-    const labelDisplay = label ? `[${label}] ` : '';
-
     if (recordId && title) {
       return (
-        <Link to={`/literature/${recordId}`}>
-          {labelDisplay} <Latex>{title}</Latex>
+        <Link className="f5" to={`/literature/${recordId}`}>
+          <Latex>{title}</Latex>
         </Link>
       );
+    } else if (title) {
+      return <span className="f5">{title}</span>;
     }
-    return `${labelDisplay}${title}`;
+    return null;
   }
 
   render() {
     const { reference } = this.props;
     const publicationInfo = reference.get('publication_info');
+    const misc = reference.get('misc');
     return (
       <List.Item>
-        <List.Item.Meta
-          title={ReferenceItem.renderTitle(reference)}
-          description={
-            <Fragment>
-              <AuthorList
-                recordId={reference.get('control_number')}
-                authors={reference.get('authors')}
-              />
-              <PublicationInfoList
-                publicationInfo={publicationInfo}
-                labeled={false}
-              />
-            </Fragment>
-          }
-        />
+        <Row gutter={24} type="flex" justify="space-around" align="middle">
+          <Col>{ReferenceItem.renderLabel(reference)}</Col>
+          <Col>
+            <List.Item.Meta
+              title={ReferenceItem.renderTitle(reference)}
+              description={
+                <Fragment>
+                  {misc && <span>{misc}</span>}
```

Comment:
I'd move this to a function

Reply:
also `misc` will appear whenever it exist which might not be what we want.

Reply:
good point, it's already solved in the serializer.